File: sandbox/ruby/src/TeamCreateDefaultExample.rb
require "dropbox-sign"

Dropbox::Sign.configure do |config|
end

team_create_request = Dropbox::Sign::TeamCreateRequest.new
team_create_request.name = "New Team Name"

begin
    response = Dropbox::Sign::TeamApi.new.team_create(
        team_create_request,
    )

    p response
rescue Dropbox::Sign::ApiError => e
    puts "Exception when calling Team#team_create: #{e}"
end

File: sandbox/php/src/TeamCreateDefaultExample.php
<?php

namespace Dropbox\SignSandbox;

require_once __DIR__ . '/../vendor/autoload.php';

use Dropbox;

$config = Dropbox\Sign\Configuration::getDefaultConfiguration();

$team_create_request = (new Dropbox\Sign\Model\TeamCreateRequest())
    ->setName("New Team Name");

try {
    $response = (new Dropbox\Sign\Api\TeamApi(config: $config))->teamCreate(
        team_create_request: $team_create_request,
    );

    print_r($response);
} catch (Dropbox\Sign\ApiException $e) {
    echo "Exception when calling Team#teamCreate: {$e->getMessage()}";
}
